package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Why there is package.json in root project? isn't it should be in either backend or learn-arai?